src/content/en/2019/http2.md

```diff
@@ -54,13 +54,18 @@ This whistle-stop tour of HTTP/2 gives the main history and concepts of the newi
 ## Adoption of HTTP/2
 As mentioned above, internet protocols are often difficult to adopt since they are ingrained into so much of the infrastructure that makes up the internet. This makes introducing any changes slow and difficult. IPv6 for example has been around for 20 years but has [struggled to be adopted](https://www.google.com/intl/en/ipv6/statistics.html).
 
+<figure>
+  <div class="big-number">95%</div>
+  <figcaption>Figure 1. The percent of global users who can use HTTP/2.</figcaption>
+</figure>
+
 HTTP/2 however, was different as it was effectively hidden in HTTPS (at least for the browser uses cases), removing barriers to adoption as long as both the browser and server supported it. Browser support has been very strong for some time and the advent of auto updating *evergreen* browsers has meant that an estimated [95% of global users now support HTTP/2](https://caniuse.com/#feat=http2).
 
 Our analysis is sourced from the HTTP Archive, which tests approximately 5 million of the top desktop and mobile websites in the Chrome browser. (Learn more about our [methodology](./methodology).)
 
 <figure markdown>
-[![Timeseries chart of HTTP/2 usage showing adoption at 55% for both desktop adnd mobile as of July 2019. The trend is growing steadily at about 15 points per year.](/static/images/2019/20_HTTP_2/http2usage.png)](/static/images/2019/20_HTTP_2/http2usage.png)
-<figcaption>Figure 1. HTTP/2 usage by request. (Source: <a href="https://httparchive.org/reports/state-of-the-web#h2">HTTP Archive</a>)</figcaption>
+[![Timeseries chart of HTTP/2 usage showing adoption at 55% for both desktop adnd mobile as of July 2019. The trend is growing steadily at about 15 points per year.](/static/images/2019/20_HTTP_2/ch20_fig2_http2_usage_by_request.png)](/static/images/2019/20_HTTP_2/ch20_fig2_http2_usage_by_request.png)
```

Comment:
`<figure markdown>` always seems to inject too many paragraph tags and the spacing is inconsistent compared to other figures. So I've been writing out the anchor and image HTML manually per the example in #237.

cc @mikegeyser

Reply:
Will take a look first thing in the morning.

Reply:
Will take a look first thing in the morning.

Reply:
Noticed that when HTML validating this.

@rviscomi want me to do your alternative way in the meantime as part of this PR?

Reply:
@bazzadp perhaps that’s for the best? We should err on the side of being conservative.

Reply:
Np. I’m kinda biased for this chapter and want it to turn out the best it can 😉

Reply:
There is a small typo above, ‘desktop adnd mobile’.

Reply:
There is a small typo above, ‘desktop adnd mobile’.

Reply:
That's done now. Didn't help as tables are also affected:

https://github.com/HTTPArchive/almanac.httparchive.org/blob/058c679a8412a551e98c6ef05a455bd8d4c19cac/src/templates/en/2019/chapters/http2.html#L231-L237

Don't really want to change all them :-(

It's definitely the wrap_tables call that's doing it as commenting that out fixes it.

A quick fix is to strip the blank ones out:

```
  body = generate_figure_ids(body);
  body = wrap_tables(body);
  body = body.replace(/<p><\/p>/g,"");
  const toc = generate_table_of_contents(body);
```

Seems safe enough. Want me to add that to this commit?

Reply:
OK I've done the following:

1. Reverted the change from markdown to HTML for images (since it's still a problem for tables so no point putting a half fix in).
2. Added an additional commit for the typo above (as that was also reverted).

I have **not** made the change to generate_chapter.js above. I think we should, but maybe not part of this commit.

So I think this is good to merge? I'm out tonight so will be back online tomorrow if any questions.